desc.en.hocon

```diff
@@ -7980,13 +8407,16 @@
       "label": "Alarm Actions"
     },
     "alarm_size_limit": {
```

Comment:
The alarm_size_limit, alarm_validity_period updates are not found in desc.zh.hocon